### PR TITLE
newfile structure

### PR DESCRIPTION
**Pull request by excellencetechnologies**

### CODE REVIEW

File: login/loginctrl.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+(function () {
```
Comment (deepakexcel) [inline login/loginctrl.js]: create a service for local storage and use $localStorage instead of localStorage

File: registration/registration.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    });
+        };
+        $scope.ifsignin = function () {
+            if (localStorage.getItem('firstname') === null && localStorage.getItem('lastname') === null && localStorage.getItem('access_token') === null) {
+
+            } else {
+                $scope.signinshow = true;
+                $scope.formshows = {"display": "none"};
+                $scope.text1 = "you have to log out first";
+                $location.path('/dashboard');
```
Comment (deepakexcel) [inline registration/registration.js]: use $state

File: index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        <!--dropdown css external file-->
+        <link href="style/style.css" rel="stylesheet" type="text/css"/>       
+        <script src="dashboard/dashboard.js" type="text/javascript"></script>
+        <script src="login/loginctrl.js" type="text/javascript"></script>
+        <script src="registration/registration.js" type="text/javascript"></script>
+        <script src="directive.js" type="text/javascript"></script>
+        <script src="service.js" type="text/javascript"></script>
+    </head>
+
+    <body class="container">
```
Comment (deepakexcel) [inline index.html]: header would come at dashboard page

Comment (excellencetechnologies) [inline index.html]: ok sir

**Pull request closed without merging**